docs/api.rst

```diff
@@ -0,0 +1,12 @@
+Api reference
+===============================
```

Comment:
Try to keep the section headers (==== etc) only as long as the above string, it looks nicer IMNSHO